### ISSUE TITLE
18. Sum Time - Splat Methods

### ISSUE THREAD

**Issue by ghost** [adebayo-dealdey/RubyTrack#18]
Modify the sum of times method for any number of time values passed to this method.
Eg: ("11:23:07","22:53:45","0:23:23","23:45:56") -> "2 day & 10:26:11"      ("11:23:07") -> "11:23:07"

https://github.com/adebayo-dealdey/RubyTrack/tree/master/SumTimes

**Comment by segunadeleye** [comment]
- Correct the definition of class methods.
- Remove comments if they are not needed.
- Why have you used `total_seconds = Integer(0)`?
- Line 13: Let the condition be on displaying an `s` or not. `1` should not be hard-coded in the result. This should be got from variable `days`.
- Save the regex in a constant.
- Why have you created the `String` class? Why do we have the same method defined in `Time` and `String`?
- In main.rb, why the conversion of `time` to string? Is this needed? Likewise in time.rb
- I got the error message 'You just entered an invalid time string, try again...' after I entered `q`.
